config/settings/development.yml

```diff
@@ -16,6 +16,7 @@ features:
     school_direct_tuition_fee: true
     pg_teaching_apprenticeship: true
     provider_led_undergrad: true
+    opt_in_undergrad: true
```

Comment:
Can we add to here please 

https://github.com/DFE-Digital/register-trainee-teacher-data/blob/2586-s-provider-led-ug-add-the-relevant-bursary/config/settings.yml#L48-L60

Reply:
it's already on master